digital_land/datatype/wkt.py

```diff
@@ -166,6 +169,10 @@ def normalise_geometry(geometry, simplification=0.000005):
     # ensure geometry is a MultiPolygon
     geometry = make_multipolygon(geometry)
 
+    if geometry:
+        if not geometry.is_valid:
+            geometry = geometry.buffer(0)
```

Comment:
What is the purpose of this?

Reply:
make_multipolygon() function does introduce issues like self-intersection in a valid geometry
Buffer attempts to fix any invalidities in the resulting MultiPolygon similar to make_valid()